charts/plugin-br-pix-direct-jd/templates/plugin-br-pix-direct-jd/configmap.yaml

```
@@ -25,15 +25,15 @@ data:
   API_RATE_ONBOARD_WINDOW: {{ .Values.pix.configmap.API_RATE_ONBOARD_WINDOW | default "1" | quote }}
 
   # JOBS
-  JOBS_CRON_TRANSACTIONS: {{ .Values.pix.configmap.JOBS_CRON_TRANSACTIONS | default "*/10 * * * *" | quote }}
+  JOBS_CRON_TRANSACTIONS: {{ .Values.pix.configmap.JOBS_CRON_TRANSACTIONS | default "*/11 * * * *" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)